src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -73,6 +74,17 @@
   margin: 0;
 }
 
+.impactTitle {
+  color: var(--color-sheltertech-blue);
+}
+
+.subtitle {
+  font: var(--font-body-medium);
+  grid-column: 2 / span 10;
+  margin: 20px 0 0;
+  overflow-wrap: break-word;
```

Comment:
Out of curiosity, what led you to setting this property? It looks like the only difference between this and the default setting of `normal` is that this will allow line breaks to be inserted in the middle of words. The [examples on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) only really demonstrate this on very narrow components, but I think that's unlikely to occur for the subtitle of the StatsBlock component, which is generally quite wide.

Although I agree that `break-word` is the more sensible default value for most text, my general approach with CSS is to set the minimum number of properties as possible to avoid adding extra complexity to the page.

Reply:
I tested the subtitle with a really long word and added the `overflow-wrap` property since the text did not wrap and would leak out of the container but I can take this out since, as you mentioned, it's unlikely to occur.